lib/Interpreter/CppInterOp.cpp

```diff
@@ -2574,26 +2580,31 @@ namespace Cpp {
     } // namespace
       // End of JitCall Helper Functions
 
-  CPPINTEROP_API JitCall MakeFunctionCallable(TCppConstFunction_t func) {
-    auto* D = (const clang::Decl*)func;
-    if (!D)
-      return {};
+    JitCall MakeFunctionCallableImpl(TInterp_t I, TCppConstFunction_t func) {
```

Comment:
```suggestion
    static JitCall MakeFunctionCallableImpl(TInterp_t I, TCppConstFunction_t func) {
```

Reply:
I need to call this function in `CXCppInterOp.cpp`.